tests/robustness/model/history.go

```diff
@@ -295,11 +295,15 @@ func (h *AppendableHistory) AppendDefragment(start, end time.Duration, resp *cli
 		h.appendFailed(request, start, err)
 		return
 	}
+	var revision int64
```

Comment:
adding revision to defragmentResponse is nice to have, not critical for this PR. I that correct? Just trying to understand

Reply:
It is based on other code changes due to some shortcuts I took. By separating models I separate `EtcdResponse` and `EtcdNonDeterministicResponse` which means that now `EtcdResponse` needs to have accurate `Revision`, which means that I need to make sure that defragmentResponse returns response with accurate revision.